include/silo/storage/metadata_store.h

```diff
@@ -13,46 +13,45 @@
 #include "silo/common/nucleotide_symbols.h"
 #include "silo/config/database_config.h"
 #include "silo/roaring/roaring_serialize.h"
-#include "silo/storage/column/date_column.h"
-#include "silo/storage/column/int_column.h"
-#include "silo/storage/column/pango_lineage_column.h"
-#include "silo/storage/column/string_column.h"
 
 namespace silo {
-
 class PangoLineageAliasLookup;
 
+namespace storage::column {
+struct DateColumnPartition;
+struct IndexedStringColumnPartition;
+struct IntColumnPartition;
+struct PangoLineageColumnPartition;
+struct StringColumnPartition;
+}  // namespace storage::column
```

Comment:
Why is a forward declaration possible here? I'd expect that the compiler has to know the size of those classes to determine the size of the fields.